src/main/java/spoon/reflect/visitor/CommentHelper.java

```diff
@@ -113,7 +109,7 @@ static void printJavaDocTag(PrinterHelper printer, CtJavaDocTag docTag, Function
 			printer.write(docTag.getParam()).writeln();
 		}
 
-		String[] tagLines = LINE_SEPARATORS_RE.split(docTag.getContent());
+		String[] tagLines = docTag.getContent().lines().toArray(String[]::new);
```

Comment:
How about making this a `forEach`?

```suggestion
		docTag.getContent().lines().forEach(line -> {
			if (docTag.getType().hasParam()) {
				printer.write(transfo.apply("\t\t"));
			}
			printer.write(line.trim()).writeln();
		});
```